wrangler.toml

```diff
@@ -21,6 +21,9 @@ DOC_REPO_NAME='prisma-cloud-docs'
 DOC_REPO_REF='main'
 DOC_REPO_ROOT_PATH=''
 BASE_PATH='/prisma/prisma-cloud'
+PREVIEW_UPSTREAM='https://raw.githubusercontent.com'
```

Comment:
@maxakuru I wasn't sure if it's needed to duplicate the vars for prod and dev ?

Reply:
yes, but only because the default `vars` are used for local development variables and `env.production.vars` are used for the deployed worker

wrangler doesn't collapse environment vars so there's always some dupes doing it this way